### ISSUE TITLE
Iteration 2

### ISSUE THREAD

**Issue by dominictarr** [dominictarr/feedopensource#20]
Okay, I wont know what is actually going in this until users have seen the result of iteration 1.
I'm only creating this to test the code that detects iteration states.

[![progress bar](https://feedopensource.com/iteration/dominictarr/feedopensource/18oBEV9hFdDMEct9Ys2bSAuEY4bi2KSMHi.png?rrrrr#1.2)](https://feedopensource.com/iteration/dominictarr/feedopensource/18oBEV9hFdDMEct9Ys2bSAuEY4bi2KSMHi#1.2)

to fund: `18oBEV9hFdDMEct9Ys2bSAuEY4bi2KSMHi`
- https://github.com/dominictarr/feedopensource/issues/40 (spending from mobile is more convienent for hot wallets)
- cache properly https://github.com/dominictarr/feedopensource/issues/46 (make the github api client code more intelligent, storing stuff locally, and requesting individual issues to confirm their state)
- decouple payment confirmation, to make it easy to use alternative currencies, such as DOGE or maybe even usd (https://github.com/dominictarr/feedopensource/issues/9#issuecomment-33108381)
- UI changes to badge https://github.com/dominictarr/feedopensource/issues/33
- UI changes to summary page: https://github.com/dominictarr/feedopensource/issues/41
- put target on badge: https://github.com/dominictarr/feedopensource/issues/48
- correct progress on iteration page: https://github.com/dominictarr/feedopensource/issues/49

anything else?

**Comment by dominictarr** [comment]
linked to issues for next iteration.

**Comment by Raynos** [comment]
The iteration 2 badge is showing 5 ticks for 5 OPEN issues. I think thats a bug.

**Comment by dominictarr** [comment]
yes.

**Comment by Raynos** [comment]
can you update the badge on this readme so its a clickable link ?

**Comment by dominictarr** [comment]
@Raynos it is now clickable.

**Comment by dominictarr** [comment]
added: show target in badge

**Comment by jhs** [comment]
https://blockchain.info/tx/a344749d6aff42703e943d6fdeef7b9e622c9b7552747b7713db242da5965edd

**Comment by Raynos** [comment]
https://blockchain.info/tx/7e334ddcd649482e1f1672e09aaf2ec4f1ffe01e4c03642f86a7f4ef3ae5dc55

**Comment by dominictarr** [comment]
added "correct progress on iteration page"

**Comment by ralphtheninja** [comment]
https://blockchain.info/tx/d46aedaf368f81e50137ff8771a5b49e9e2085cec0637f09b8027dc990224204

**Comment by ralphtheninja** [comment]
There seem to be some problems with the badge. It's not visible in this issue.

**Comment by Raynos** [comment]
If you go to the badge directly it works, it looks like githubs https cache failed :/

**Comment by dominictarr** [comment]
github changed how images work :(

**Comment by dominictarr** [comment]
okay, I've fixed the headers... and changed the url, but it's not working for some reason.
I've opened an issue here: https://github.com/atmos/camo/issues/37

**Comment by dominictarr** [comment]
the badge is working again. the problem was that I didn't know to provide the cert chain to the server. it's that is not in the node https docs!

**Comment by Raynos** [comment]
:)

**Comment by RobbieClarken** [comment]
https://blockchain.info/tx/1a0fe73e0ac38d96f9f146fcc843abefced06a3cad29ebf12be89a4a54993645

**Comment by joates** [comment]
fyi: i'm going to a btc meetup in London on Feb 3rd, gonna buy some coin 4 ca$h because i wanna help fund this 1 ( _plus i can get on the funders wall of fame_ !!) \o/ yay

**Comment by ralphtheninja** [comment]
@joates Cool! Is that the meetup that @aantonop is attending?

**Comment by joates** [comment]
@ralphtheninja yes, he is scheduled, also Dr Gavin Wood (Ethereum core developer) probably 50% of the info will go _way over_ my head, the other half will go _way way over_ !! ;-) but i got to start learning this stuff at some point, might as well be now :))

**Comment by joates** [comment]
> anything else?

@dominictarr this is _way off-topic_ but..

the prospect of Ethereum script (p̶y̶t̶h̶o̶n̶ ̶d̶e̶r̶i̶v̶a̶t̶i̶v̶e̶ [edit]it's actually a [turing complete](http://en.wikipedia.org/wiki/Turing_completeness#Formal_definitions) VM, so not limited to python![/edit]) making it possible to actually embed the application into the blockchain transaction has completely blown my mind these past few days.

this is a rapidly evolving technology & i think we just need to be aware of whats on the horizon and consider how it might affect this initiative as it emerges. ( _IMO ethereum will be very well funded --when it launches-- and be very quick to get the product into the marketplace, there is already a c++ client but i guess it's pre-beta_ ).

**Comment by joates** [comment]
https://blockchain.info/tx/532ae78faea5db6e1aa9853493618db601e1bb01aa3e4c37751448d114ebeeb2

**Comment by binarykitchen** [comment]
https://blockchain.info/tx/e19608d824601343af871a254fd9ce33ff8a07a2d1f1322bdff4c2230330e69e

**Comment by clehner** [comment]
https://blockchain.info/tx/8d5ee79f2aae765a2477d252164ccc07389533abb16c236127f1d3ebf5ae2ba8

**Comment by joates** [comment]
stalled ?
@dominictarr have you considered lowering the target to 1.0btc ?
alternatively..
how do we attract the additional 0.2btc funding ?

**Comment by dominictarr** [comment]
oh, I meant to post this update:
I am gonna just start this anyway soon,
I've just gotta finish up with some other stuff I have going on.

**Comment by joates** [comment]
bd408bf50da0c1f62715579e71450a5116c4ad02d150c722a0b60778cec84565

**Comment by dominictarr** [comment]
okay, planning to start on this next week...

**Comment by dominictarr** [comment]
okay, most of the UI stuff here is fixed (except that the badge looks wrong), next is the caching stuff, which is more complex, and will fix the badge.

**Comment by joates** [comment]
i've put quite a bit of effort into supporting this project, but it is unusable in its current state, i am now exclusively using [bitcore](https://github.com/bitpay/bitcore) as my primary app dependency, ~~@dominictarr would you please consider refunding my BTC from this iteration ?~~
[edited to remove refund request, although @dominictarr offered the full refund without hesitation]

**Comment by NHQ** [comment]
@joates :-1:

**Comment by dominictarr** [comment]
@joates, I'm sorry to hear that - but I'll do that.

**Comment by dominictarr** [comment]
by the way, I've picked the low hanging fruit for this iteration - next is just dealing with the caching - since I need something very like that for npmd, I've been working on https://github.com/dominictarr/level-content-cache which I'm also using https://github.com/dominictarr/npmd-cache. So, I should be finishing off this iteration soon.

**Comment by joates** [comment]
i want to revise my previous comment..
although i have decided to implement an alternative payment solution for my projects that doesn't change the fact that i _also_ want to continue to support _this_ project.
i was just venting some frustration, please accept my sincere apologies @dominictarr

**Comment by NHQ** [comment]
@joates :+1: 

Bitcore looks nice.  I can see the potential for its use in FeedOpenSource, and obviously elsewares.  What are your thoughts on it @dominictarr? 

We gotta give FOS a breathing chance tho.  Ugly as it is currently, and struggling with the very problem it seeks to resolve.

The speed at which BTC protocol based apps and systems are being built makes me wonder if FOS should focus more on the iterations, feedback, and general "crowd" related things, etc, instead of the BTC details.  Crowdsourcing is getting crowded, in a good way.  It will be nice to get FOS up to the level of, say, crowdtilt or like.  I got some FOSS I wanna get funded!

To prove I'm serious, imma close this round funding.  

[point1](http://blockexplorer.com/tx/9b9a42e95e7d5f625f3c7b18ddd6af871a9a6273a4c07c81ac28feb89fda67a1)
[point2](http://blockexplorer.com/tx/c0c300a15049e3d3eb23db9f2ad9ab8343f4d217f52e0a26b7a39780db794330)

<3

**Comment by dominictarr** [comment]
Awesome! Thanks, I'm hopefully gonna get on this next few days - have 50% of the stuff ready.

**Comment by dominictarr** [comment]
just 2 to go. the hardest will be the caching, but it might not be that hard.
hmm - since these urls are very mutable, it will pretty much have to request them everytime,
however, if we use caching headers correctly, they do not count against github's rate limit, so they we'll be able hit them more often.

**Comment by raucao** [comment]
Any progress on this iteration since April?
